configs/zeitgeist.yml

```diff
@@ -0,0 +1,30 @@
+endpoint: wss://main.rpc.zeitgeist.pm/ws
+# endpoint: wss://zeitgeist-rpc.dwellir.com
```

Comment:
can you remove these or make it array. chopsticks supports multiple endpoints

Reply:
https://github.com/AcalaNetwork/chopsticks/pull/591/commits/c0e51254d0723eaa61ee0d130c580bcd1b72cc58